src/main/java/com/faforever/client/replay/ReplayDetailController.java

```diff
@@ -449,20 +469,53 @@ private List<TeamCardController> createTeamCardControllers(Map<String, List<Game
     }).toList();
   }
 
+  private GameOutcome calculateGameResult(Map<PlayerBean, GamePlayerStatsBean> statsByPlayer) {
+    // Game outcomes are saved since 2020, so this should suffice for the 
+    // vast majority of replays that people will realistically look up.
+    Map<GameOutcome, Long> outcomeCounts = statsByPlayer.values()
+        .stream()
+        .map(GamePlayerStatsBean::getResult)
+        .filter(Objects::nonNull)
+        .map(gameOutcome -> (gameOutcome == GameOutcome.CONFLICTING) ? GameOutcome.UNKNOWN : gameOutcome)
+        .map(gameOutcome -> (gameOutcome == GameOutcome.MUTUAL_DRAW) ? GameOutcome.DRAW : gameOutcome)
+        .collect(Collectors.groupingBy(gameOutcome -> gameOutcome, Collectors.counting()));
+
+    if (outcomeCounts.containsKey(GameOutcome.VICTORY)) {
+      return GameOutcome.VICTORY;
+    }
+
+    return outcomeCounts.entrySet()
+        .stream()
+        .max(Entry.comparingByValue()).map(Entry::getKey).orElse(GameOutcome.UNKNOWN);
```

Comment:
Sometimes not all players report victory in the team that won. I think it happens when a player dies before the end of the match, then they report "defeat" for themselves. From my empirical testing I concluded that as soon as we detect a victory, we can assume that that team has won, even if everyone else in the team reports defeat

Reply:
Wouldn't that be the case as well for draw? So does it make more sense rather than taking the mode taking the max reported result where Max is defined as priority?

Reply:
I don't really understand what you mean. Can you rephrase that?

Reply:
See if anyone reported victory, draw, defeat, unknown and take the first one because the default is everyone reports defeat so we only need to check if someone reported something other than that

Reply:
Ah yeah, I see now what you mean. I think you are correct. So that means I don't even need to count, but I just need to check the presence of results in that order.